lib/elixir/lib/option_parser.ex

```diff
@@ -605,23 +609,23 @@ defmodule OptionParser do
     end
   end
 
-  defp tag_option(option, _switches, allow_nonexistent_atoms?) do
+  defp tag_option(option, %{allow_nonexistent_atoms?: allow_nonexistent_atoms?}) do
     if option_key = get_option_key(option, allow_nonexistent_atoms?) do
       {:default, option_key}
     else
       :unknown
     end
   end
 
-  defp tag_oneletter_alias(alias, aliases, allow_nonexistent_atoms?) when is_binary(alias) do
+  defp tag_one_letter_alias(alias, %{aliases: aliases, allow_nonexistent_atoms?: allow_nonexistent_atoms?}) when is_binary(alias) do
     if option_key = aliases[to_existing_key(alias, allow_nonexistent_atoms?)] do
       {:default, option_key}
     else
       :unknown
     end
   end
 
-  defp expand_multiletter_alias(letters, value) when is_binary(letters) do
+  defp expand_multi_letter_alias(letters, value) when is_binary(letters) do
```

Comment:
"multi-letter" becomes `multi_letter` here, but "allow non-existent atoms" becomes `allow_nonexistent_atoms?`. We should remove the `-` in variable names, not replace it with `_`, so we should leave this function name unchanged. :)